### PR TITLE
Use named parameters instead of $ and % in localised strings

### PR DESCRIPTION
**Pull request by absidue**
# Use named parameters instead of $ and % in localised strings

## Pull Request Type

- [x] Bugfix

## Related issue
The codeql checks were added in #2650 (see the files tab on that pull request for the warnings).

## Description
This pull reqest aims to fix the two issues reported by codeql: "Incomplete string escaping or encoding" and "Incomplete URL substring sanitization".

I wasn't sure whether I should change all of the language files, as Vue-i18n will happily use the old strings in languages that haven't been updated as, the key didn't change for all of the strings (if the key changes, vue-i18n can't find the translated string and falls back to the en-US one, which doesn't always happen here). I can definitely change all of the strings in the left-to-right languages as part of this pull request if you want me to, I won't be able to do right-to-left languages as the text seems to change entirely when I replace the dollar ($) or percent (%) symbols, so I definitely want to leave those up to the translators on weblate.

## Testing 
Ideally check every place where the changed strings show up, only en-US and en_GB will work properly until the other languages are updated. Although that might be too much testing for you so you could just check a couple of strings.

## Desktop

- **OS:** Windows
- **OS Version:** 10
- **FreeTube version:** 0.17.1

### CODE REVIEW

File: src/renderer/views/SubscribedChannels/SubscribedChannels.js [hunk 1]
```diff
@@ -160,7 +160,7 @@
 
     thumbnailURL: function(originalURL) {
       let newURL = originalURL
-      if (originalURL.indexOf('ggpht.com') > -1) {
+      if (new URL(originalURL).hostname.endsWith('ggpht.com')) {
```
[the remark this answers is not in the document]
Comment (ChunkyProgrammer) [reply src/renderer/views/SubscribedChannels/SubscribedChannels.js:163]: We could dismiss the alert?

[the remark this answers is not in the document]
Comment (absidue) [reply src/renderer/views/SubscribedChannels/SubscribedChannels.js:163]: Seems like Invidious always converts the URLs to `yt3.ggpht.com` on the backend so maybe I can just hard code it to check for that?
[iv-org/invidious /src/invidious/routes/images.cr#L51](https://github.com/iv-org/invidious/blob/master/src/invidious/routes/images.cr#L51)

**Comment by PikachuEXE** [comment]
Should we also update other locale files or in separate PR?
Do other not updated locales work?

**Comment by absidue** [comment]
I'll update the other language files now, otherwise you'll see random `$` and `%` in FreeTube until those languages are updated.

**Review by ChunkyProgrammer — DISMISSED** [review]
Lgtm!

**Comment by PikachuEXE** [comment]
Not sure which file(s) but I got new errors in console
image

**Comment by absidue** [comment]
@PikachuEXE You can click on that link and it will show you the exact line in the code that caused the error, in this case we only use yaml-loader in one place, so I know where the error is happening. I'm guessing that some of the language files have YAML syntax errors.

**Comment by PikachuEXE** [comment]
image

**Comment by ChunkyProgrammer** [comment]
Locales with issues: it, ja, ko, nb_NO

**Comment by ChunkyProgrammer** [comment]
@absidue if you use vscode, I recommend installing  this yaml extension to find the problems in the files (looks like quotes are missing around some values) https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml

**Review by PikachuEXE — APPROVED** [review]
Tested locally briefly with zh-TW

**Review by ChunkyProgrammer — APPROVED** [review]

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Pull request merged**